externs/svg-externs.js

```diff
@@ -0,0 +1,10367 @@
+/**
+ * @fileoverview Generated by closureidl 0.2dev closureidl-parser-webkit 0.2dev dart-idl 7592.
```

Comment:
+1 for just including this file in Blockly's codebase as it is.  Every dependency adds complexity and fragility, and the SVG API is not a moving target.

Also +1 for encouraging the Closure Compiler team to add it into their default externs.  SVG is no longer the fringe and poorly supported part of the web that it once was.